app/src/lib/api.ts

```diff
@@ -361,6 +365,48 @@ export interface IAPIRefCheckRuns {
   readonly check_runs: IAPIRefCheckRun[]
 }
 
+interface IAPIWorkflowRuns {
+  readonly total_count: number
+  readonly workflow_runs: IAPIWorkflowRun[]
```

Comment:
Any reason for not using `ReadonlyArray`?
```suggestion
  readonly workflow_runs: ReadonlyArray<IAPIWorkflowRun>
```